.github/workflows/action.yml

```diff
@@ -1,21 +1,40 @@
-name: Add build and publish Bucket
+name: Build and publish to s3 bucket
 on: push
 
 jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Use Node.js 12.x
+        uses: actions/setup-node@v1
+        with:
+            node-version: 12.x
+      - run: yarn install
+      - run: curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > "./cc-test-reporter"
+      - run: chmod +x "./cc-test-reporter"
+      - run: ./cc-test-reporter before-build
+        env:
+            CC_TEST_REPORTER_ID: ${{ secrets.CC_TEST_REPORTER_ID }}
+      - run: yarn test:coverage
+      - run: ./cc-test-reporter after-build --exit-code $?
```

Comment:
We could use a cache to speed things up, but I don't now how big the reporter is, and how much of an issue it is.

Reply:
It's small go executable.
But about cache, I don't know if there a built-in cache for the yarn install (I would say no)
The build doesn't seem not too slow for the moment.

Reply:
Then I don't think we should worry about it. Less complexity is a feature :)